js/wide-tables.jquery.js

```diff
@@ -132,6 +132,23 @@ $.fn.wideTables = function wideTablesPlugin(options) {
     $container.data('id7.wide-tables', wideTables);
   }
 
+  if (!(navigator.userAgent.indexOf('Safari') !== -1 && navigator.userAgent.indexOf('Chrome') === -1)) {
+    if (navigator.platform === 'MacIntel') {
+      // on a mac we want to show the scroll bar as we cannot detect system preferences setting
```

Comment:
It's good to have some commentary here as this behaviour isn't self-evident. Could we condense the explanation down to just what this does and why?

Reply:
agreed